Automatic configuration generation

shahbaz-mehar opened the pull request.

emicha reviewed (commented):
Please add a section in the README.md explaining how to synch terminals by running this task.

Does this happen automatically when deploying the instance?

shahbaz-mehar commented:
> Please add a section in the README.md explaining how to synch terminals by running this task.
> 
> Does this happen automatically when deploying the instance?

@emicha I have added the section in the README.md file regarding how to sync terminals with the gateway. Apart from that, I have removed the extra bits from the README.md file as they are already part of readme.txt that is according to WordPress marketplace requirements. Also moved the readme-dev.txt contents to README.md file.

shahbaz-mehar commented:
Configurations are loading automatically when we deploy the instance.
